src/hardware/src/Motor.cpp

```diff
@@ -9,18 +9,23 @@ namespace Hardware{
         deviceID = ID;
     }
     void Motor::setPower(double power) {
+        const std::lock_guard lock{this->mutex};
```

Comment:
Didn't catch this in the first review but `Motor.cpp` uses `this->mutex` while `Group.cpp` uses `mutex`. Might consider changing for consistency

Reply:
Fixed in 75bf686.